proto/README.md

```diff
@@ -12,6 +12,18 @@ proto/
   sharding/
     p2p/
       v1/
+  testing/
 ```
 
-We specify messages available for p2p communication common to beacon chain nodes and sharding clients.
\ No newline at end of file
+We specify messages available for p2p communication common to beacon chain nodes and sharding clients.
```

Comment:
Can you add a few sections here about the p2p middleware stuff for visibility?

Reply:
What does this have to do with protos?